deploy/roles/python-libs/defaults/main.yml

```diff
@@ -9,7 +9,8 @@ python3_packages:
   - requests
   - cookiecutter
 
-python_pip_bin: /usr/local/bin/pip2.7
+#python_pip_bin: /usr/local/bin/pip2.7
+python_pip_bin: /usr/bin/pip2.7
```

Comment:
It's `/usr/local/bin/pip2.7` in production because of historical reasons, though if we shift to software collections that will change.